spec/unit/coinbase/wallet_spec.rb

```diff
@@ -113,6 +113,7 @@
       let(:use_server_signer) { false }
 
       before do
+        Coinbase.configuration.use_server_signer = false
```

Comment:
Instead of setting this explicitly on the actual configuration object which can cause test pollution can we mock this? 

```
let(:configuration) { instance_mock(Coinbase::Configuration, use_service_signer: use_server_signer) }

before do
  allow(Coinbase).to receive(:configuration).and_return(configuration)
end
```

And then in different test cases we can add:

```
let(:use_server_signer) { true}
```